tests/e2e/support/objects/app-admin-settings/groups/actions.ts

```diff
@@ -3,17 +3,32 @@ import { Page } from 'playwright'
 const newGroupBtn = '.admin-settings-app-bar-actions'
 const createGroupInput = '#create-group-input-display-name'
 const actionConfirmButton = '.oc-modal-body-actions-confirm'
+const groupTrSelector = 'tr'
 
-export const createGroup = async (args: { page: Page; key: string }): Promise<void> => {
+export const createGroup = async (args: { page: Page; key: string }): Promise<string> => {
   const { page, key } = args
   await page.locator(newGroupBtn).click()
   await page.locator(createGroupInput).fill(key)
 
-  await Promise.all([
+  const [response] = await Promise.all([
     page.waitForResponse(
       (resp) =>
         resp.url().endsWith('groups') && resp.status() === 200 && resp.request().method() === 'POST'
     ),
     page.locator(actionConfirmButton).click()
   ])
+  const group = await response.json()
+  return group.id
+}
+
+export const getDisplayedGroups = async (args: { page: Page }): Promise<string[]> => {
+  const { page } = args
+  const groups = []
+  const result = page.locator(groupTrSelector)
+
+  const count = await result.count()
+  for (let i = 0; i < count; i++) {
+    groups.push(await result.nth(i).getAttribute('data-item-id'))
```

Comment:
We can, but I used the existing approach (see "Users and spaces")

Reply:
```
When "Admin" creates the following groups
      | id       |
      | sales    |
      | security |
 Then "Admin" should see the following group
      | group    |
      | sales department    |
      | security department |
```

Reply:
Hmmm :thinking:, I would have done like this:
```feature
 When "Admin" creates the following groups
      | group   |
      | mygroup |
 Then "Admin" should see the following group
      | group   |
      | mygroup |
```
and would not have used `groupStore`.
- create a group (any name)
- check for the displayed group name (if possible)

![Screenshot from 2023-03-21 15-35-52](https://user-images.githubusercontent.com/52366632/226571414-d8c04e7f-33e0-486f-bac3-3ec8b4d5be67.jpg)

Reply:
> We can, but I used the existing approach (see "Users and spaces")

Yeah, we could have done better there. If we are to change this behavior then we can do it in other PRs. Not to be done in this PR.

Reply:
Or, are there any specifics that we have to check the ids that are in the element attributes? IMO, we should test only what is displayed in order to best mimic the real user

Reply:
Ahh, i think we started to use this approach when we check spaces (spaces can have same name) or do we have another reasons?

I'll change my PR. Thanks for suggestions

Reply:
to run test twice we need delete group before

Reply:
Yeah, since we don't clear up groups/spaces/user after the scenario, this looks to be a valid point. Maybe we can think about clearing data created (during the test) after every scenario. I have no idea why we don't have implemented this teardown approach as we did in other tests. I think @SwikritiT is doing something related to teardown the spaces created during the tests. But that's not to discuss in this PR.

Reply:
@saw-jan @ScharfViktor I created a issue for this https://github.com/owncloud/web/issues/8662